index.html

```diff
@@ -23,8 +23,8 @@
     <section class="intro-wrapper">
         <section class="welcome">
```

Comment:
Kindly. the semantic SECTION tags are used to identify each theme of your page, I recommend you use them like this, inside them, you can put the necessary content, but putting a SECTION inside another SECTION is not a good practice, I recommend changing it to a DIV.

Reply:
Thanks for the tip. It has been incorporated 👍